app/presenters/two_factor_auth_code/max_attempts_reached_presenter.rb

```diff
@@ -29,15 +28,6 @@ def troubleshooting_options
       [read_about_two_factor_authentication, contact_support]
     end
 
-    def js
```

Comment:
was this not being called anywhere?

Reply:
It is, through a bit of magic in the view:

https://github.com/18F/identity-idp/blob/157be3b85a915f2d5e92e2c5d1c0aca3152eb0d6/app/views/shared/_failure.html.erb#L19-L21

Reply:
ooof, inheritance 😭 

Looks like this was the last instance of that being called with something non-empty, so we can remove all that?

```
> git grep "def js"
app/presenters/failure_presenter.rb:  def js; end
app/presenters/two_factor_auth_code/max_attempts_reached_presenter.rb:    def js
```

Reply:
Removed `js` in [c3713ad](https://github.com/18F/identity-idp/pull/6129/commits/c3713adc3e84666e1d62d0abc15dbbb31e225526).